app/components/elements/DatePicker.js

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import momentPropTypes from 'react-moment-proptypes';
+import { SingleDatePicker, SingleDatePickerShape } from 'react-dates';
+import NavigateBeforeRoundedIcon from '@material-ui/icons/NavigateBeforeRounded';
+import NavigateNextRoundedIcon from '@material-ui/icons/NavigateNextRounded';
+import CloseRoundedIcon from '@material-ui/icons/CloseRounded';
+import omit from 'lodash/omit';
+import noop from 'lodash/noop';
```

Comment:
I like the specific function imports on `lodash`

Reply:
Me too :)